### ISSUE TITLE
Make the terrain moveable

### ISSUE THREAD

**Issue by Zylann** [Zylann/godot_terrain_plugin#10]
The terrain node could inherit Spatial, making it more straightforward to find and transform in the scene.
This must be profiled because there are lots of meshes under it.

**Comment by Gibbz** [comment]
yep, this is pretty important for making a level which requires multiple terrains. 
Is the project still being developed?

**Comment by Zylann** [comment]
I'm not working on it at the moment. Both because I have other projects in progress, and also because I feel a GDScript terrain plugin such as this one hits limitations too quickly due to the slowness of GDScript, and it also hits my motivation to improve it (while a C++ module would be perfect). Some things are still doable though.
